README.md

```diff
@@ -1,16 +1,235 @@
+managed to save 100 of the trials to the file `trials.csv`.
+
+The rows in the CSV represent:
+
+```csv
+1,Arya,6,"4,3"
+```
+
+1.  An `id` for the trial
+2.  A participant name (a candidate)
+3.  The maximum number of pips on the die thrown in the next field
+4.  The results of the candidate's throw. Multiple die are separated by a `,`
+
+So the line above says, in English, "In trial 1, Arya threw six sided dice.
+There were two of them (splitting the last field and counting the numbers),
+their pips were 4 and 3 leading to a total pip-value of `7`."
+
+### First Release: Learn About the Data
+
+We want to analyze this file for the DoLS.
+
+The tests will guide you in creating a class called `LuckAnalyzer.rb` in
+`lib/luck_analyzer.rb`.
+
+- `LuckAnalyzer` class will:
+  - Be initialized by being passed the name of the file to read. It must also
+    end in `.csv`. Else we `raise` an `ArgumentError`
+  - `initialize`
+    - Will build a file path from the file name passed in (starter code is
+      provided)
+    - Will "parse" the data in the file at the file path according to the CSV
+      format using the Ruby [CSV] library method that reads in a file and
+      converts the rows to an `Array` of `Arrays`. Consult the
+      [documentation][csv] for this method
+    - The parsed CSV data should be available through a reader called
+      `csv_data`
+  - Instance methods
+    - `#common_number_of_trials`: Because the database crashed, we're not sure
+      how many trials are common to the participants. We need this class to
+      tell us the minimum common number of trials. For example: [A, A, A, A, B,
+      B, B, B, B, C, C, C] has a common trial of `3` since C has the fewest.
+    - `#least_trials_candidate`: Which candidate had the least, and thus
+      common, number of trials? Returns a `String`
+    - `#most_trials_candidate`: Which candidate had the most trials? Returns a
+      `String`
+
+> **TYPOGRAPHICAL NOTE**: Developer documentation uses `#methodname` to show that
+> `methodname` is an instance method. They use `::methodname` to denote a class
+> method. For trigonometry equations there's `Math::sin`, for instances of
+> `Mankind` there's `Mankind#sin`.
+
+Use the command `rspec spec/luck_analyzer_spec.rb` from the terminal and get
+all of these tests passing.
+
+### Second Release: Who is the Luckiest?
+
+The DoLS would like for you to determine the following:
+
+1.  "Who is the luckiest candidate?" Where:
+2.  "Lucky" is defined as having had a dice roll where the total pips is equal
+    to `7`
+3.  "Luckiest" is defined as having the most "lucky" rolls within a set of
+    size `common_number_of_trials`. We should take as many "lucky" rolls as
+    possible
+    - Example
+    - Gven a common size of 3 (where "L" means "Lucky" and "U" means "Unlucky"):
+    - A has L, L, L, L => [L, L, L] => 3/3 => 100%
+    - B has U, U, U, L => [L, U, U] => 1/3 => 33%
+    - C has U, U, U, U, U, U, U, U, U, L, U, L => [L, L, U] => 66%
```

Comment:
I'm not clear on what determines the sample size of 3 here and students may think `- B has U, U, U, L => [U, U, U]` since there are three `U`s first

Reply:
The 3 is the result of running `#common_number_of_trials`.

Reply:
Updated in 9bc3ecf @maxwellbenton ?

Reply:
Or, improved, rather?